srml/system/src/lib.rs

```diff
@@ -77,7 +77,6 @@
 //! 	pub struct Module<T: Trait> for enum Call where origin: T::Origin {
 //! 		pub fn system_module_example(origin) -> Result {
 //! 			let _sender = ensure_signed(origin)?;
-//! 			let _random_seed = <system::Module<T>>::random_seed();
```

Comment:
This example now makes no sense. It should be reworked. 

Either use something else from `System` to demonstrate something that it can do (easier) or `pub trait Trati: system::Trait + randomness::Tratit` and do the same.